### ISSUE TITLE
[BUG] Let's encrypt renew did not run from cron

### ISSUE THREAD

**Issue by docop** [Zer0CoolX/guacamole-install-rhel#115]
Get few test server deploy with your script From a centos7.6 , centos7.7.. with script from november and from the feb2020 script. All default install, default password as per your gif animation. Only change: it's install with the :Let's Encrypt.

 So not sure about the cron job setting.. but it never do the certificat renew from several install and also from a past one a year ago.
But if i log manually and do : sudo certbot renew --dry-run   , it show work
and when i do : sudo certbot renew   . it work and renew fine.

 Not sure what can be the proper setting for the cron job or so..
thanks

**Comment by docop** [comment]
As a note, i did received all the email from let's encrpyt about the certificat will be expire. And other when it's expired. But this morning i did log and : certbot renew , in a server with few day left before expiry.

**Comment by Zer0CoolX** [comment]
I would start by checking your cronjobs/service. Its possible some of the scripts you used are from before the script was changed to utilize this.

The lines in the script that setup the renewal are:

```
systemctl enable certbot-renew.service
systemctl enable certbot-renew.timer
systemctl list-timers --all | grep certbot
```

Check that they exist, are enabled, etc.

You mention you checked a log, but not what log. Check /var/log/messages. Lets Encrypt may also have its own log, if so check it.

Make sure you have ports 80/443 forwarded from your gateway/router/firewall to the Guacamole server. Lets Encrypt requires port 80 to get and renew certs.

Without going back and checking commits myself, I cannot say when I added the certbot/renewal to the script. Also I generally cannot support older versions of my script for that very reason. I hope the above helps
